tensorboard/webapp/notification_center/_views/notification_center_component.scss

```diff
@@ -12,3 +12,53 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+
+::ng-deep .notification-menu.mat-menu-panel {
+  width: 350px;
+  font-size: 10px;
+}
+
+.menu-item {
+  padding: 15px 15px 0 15px;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.1);
+}
+
+.menu-item:last-child {
+  border-bottom: 0;
+}
+
+.notification-menu {
+  font-size: 10px;
+}
+
+.category-icon {
+  height: 15px;
+  vertical-align: middle;
+  width: 15px;
+  color: #4285f4;
+  margin-right: 6px;
+}
+
+.content-wrapper {
+  margin-left: 20px;
+  margin-top: 15px;
+  line-height: 15px;
+}
+
+.title {
+  margin: 15px 0 0 0;
+  line-height: 15px;
```

Comment:
what is the font-size? Will this result in awkward font clipping for CJK characters? Prefer to make it a multiple of a font-size like `1.4`